setup.py

```diff
@@ -317,7 +317,7 @@ def do_setup():
             'requests>=2.20.0, <3',
             'setproctitle>=1.1.8, <2',
             'sqlalchemy>=1.1.15, <1.3.0',
-            'tabulate>=0.7.5, <=0.8.2',
+            'tabulate>=0.7.5, <=0.8.3',
```

Comment:
Should we change it to `>=0.7.5, <0.9` instead?

Reply:
@ashb we can. According to their repo the next version will be 0.8.4